src/pages/access_review/AccessReviewHeader.js

```diff
@@ -16,10 +17,14 @@ const SMALL = {
 };
 
 export const AccessReviewHeader = hh(class AccessReviewHeader extends React.PureComponent {
-
-  openAccessReview = (referenceId) => {
-    this.props.history.push(`/access_review/${referenceId}`);
-  };
+  constructor(props) {
+    super(props);
+    if (!fp.isNil(props.dacChairMessage)) {
+      this.state = { dacChairMessage: props.dacChairMessage };
+    } else {
+      this.state = { dacChairMessage: "" };
+    }
+  }
```

Comment:
Instead of making this a state variable, you can choose not to render the message at all (which is preferable) based o the existence of the prop. A general principle in React is to avoid setting state when something can be easily calculated from existing props.

Reply:
Secondarily, the naming of this, now that it is extracted into a prop, could be more generic. Something like, `message`